db/migrate/20200907133359_create_companies.rb

```diff
@@ -0,0 +1,10 @@
+class CreateCompanies < ActiveRecord::Migration[6.0]
+  def change
+    create_table :companies do |t|
+      t.string :name,   null: false, default: ""
+      t.string :domain, null: false, default: ""
+      t.timestamps
```

Comment:
null: false, change :domain to :subdomain and remove default if not required

Reply:
done